lib/vintage_net_wizard/backend/default.ex

```diff
@@ -148,13 +148,10 @@ defmodule VintageNetWizard.Backend.Default do
   def handle_info(_, state), do: {:noreply, state}
 
   defp apply_configurations(wifi_configurations, state) do
```

Comment:
I'm confused on the type for `wifi_configurations` here. It look like it is spec'd to be a `WiFiConfiguration.configuration()`, but the usage is a `VintageNetWiFi` network configuration map. Is this wrong or is `WiFiConfiguration.configuration()` a spec for the networks maps from `VintageNetWiFi`?